packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1015,6 +1020,7 @@ class CupertinoModalPopupRoute<T> extends PopupRoute<T> {
     bool? semanticsDismissible,
     ImageFilter? filter,
     RouteSettings? settings,
+    this.anchorPoint,
```

Comment:
This does not appear tested directly, nor through the use of showCupertinoModalPopup